pym/bob/cmds/jenkins.py

```diff
@@ -848,6 +848,11 @@ def ensureFingerprint(step):
                         AUDIT=JenkinsJob._auditName(d),
                         WSP_PATH=d.getWorkspacePath()))
 
+        # remove @tmp directory generated by some jenkins plugins
+        for d in sorted(self.__checkoutSteps.values()):
+            prepareCmds.append("# remove @tmp directories created by some jenkins plugins")
```

Comment:
The comment should probably be added only once...

Reply:
But you still add the comment for each SCM again...

Reply:
now I got you point :sleepy: